data-tiles/geo-2021.mk

```diff
@@ -0,0 +1,139 @@
+#
+# geo-2021.mk -- rules to download and process 2021 geographies
+#
+# This makefile is mean to be included by GNUmakefile; it's not independent
+#
+# Variables this file is expected to set
+#
+#	GEO_DOWNLOADS	list of downloaded geo files
+#	GEO_PROCESSED	list of processed geo files
+#
+# Each file named in those variables should have targets in this file.
+
+#
+# URLs for raw geojson files
+#
+
+LAD_URL=https://opendata.arcgis.com/api/v3/datasets/d31f826e318d441390f54f472d976ee1_0/downloads/data?format=geojson&spatialRefId=4326&where=1%3D1
+
+LSOA_URL=https://opendata.arcgis.com/api/v3/datasets/8fb6ac064a704cee9a999ee08414d61e_0/downloads/data?format=geojson&spatialRefId=4326&where=1%3D1
```

Comment:
NB - LSOA aren't currently part of the atlas, and I'm not sure if there are any plans to include them any time soon?